contrib/aleph-traefik-minio-keycloak/docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
       - "traefik.enable=false"
 
   ingest-file:
-    image: ghcr.io/alephdata/ingest-file:3.18.3-rc1
+    image: ghcr.io/alephdata/ingest-file:latest
```

Comment:
Isn't this going to pull down a 3.18.x version of ingest? Don't we want to have 3.17.1 as a pegged version right now?

Reply:
It is. My thinking was that it is annoying to bump these every time we release something. I think we should make sure that `latest` always points to something sensible, so perhaps we should tag the `3.17.1` as `latest` for now?

Reply:
I get it. But when it pulls latest will it pull RC versions? Not sure I'd be comfortable with that.

Reply:
You are totally right, I didn't think of that. I will revert that. Thanks!